src/cli/templates.json

```diff
@@ -0,0 +1,28 @@
+{
+    "templates": [
+        {
+            "title": "Daily Image Reveal Game",
+            "value": "github:playpassgames/daily-pixel-game-template"
+        },
+        {
+            "title": "Daily Images Game",
+            "value": "github:playpassgames/daily-image-game-template"
+        },
+        {
+            "title": "Daily Level Game",
+            "value": "github:playpassgames/playpass-game-template"
+        },
+        {
+            "title": "Daily Phrase Game",
+            "value": "github:playpassgames/daily-phrase-game-template"
+        },
+        {
+            "title": "Daily Word Game",
+            "value": "github:playpassgames/daily-word-game-template"
+        },
+        {
+            "title": "Daily Audio Game",
+            "value": "github:playpassgames/daily-audio-game-template"
+        }
+    ]
+}
```

Comment:
I like pulling this out into a different source file, but can we have it be in .ts instead of .json? JSON isn't type checked, and is annoying to edit (no comments, strict quotes and commas).

Reply:
I made it a json in hopes of it being easier to allow third parties to contribute their templates that we want to approve and promote with the SDK.  Json files are usually treated as "not code" so some may be less hesitant to edit and contribute to it.